src/nft/components/bag/BagFooter.tsx

```diff
@@ -221,12 +236,16 @@ export const BagFooter = ({
                 <Trans>Total</Trans>
               </ThemedText.SubHeaderSmall>
               <ThemedText.HeadlineSmall>
-                {formatWeiToDecimal(totalEthPrice.toString())}&nbsp;{activeCurrency?.symbol ?? 'ETH'}
+                {inputCurrency
+                  ? swapState !== TradeState.VALID
+                    ? '-'
+                    : ethNumberStandardFormatter(swapTrade?.inputAmount.toExact())
+                  : formatWeiToDecimal(totalEthPrice.toString())}
+                &nbsp;{activeCurrency?.symbol ?? 'ETH'}
               </ThemedText.HeadlineSmall>
-              <ThemedText.BodySmall color="textSecondary" lineHeight="20px">{`${ethNumberStandardFormatter(
-                totalUsdPrice,
-                true
-              )}`}</ThemedText.BodySmall>
+              <ThemedText.BodySmall color="textSecondary" lineHeight="20px">
+                {`${ethNumberStandardFormatter(totalUsdPrice, true)}`}
```

Comment:
Looks like there's discrepancy between stable swap inputAmount and totalUsdPrice. We may want to change our usd price fetcher to use the same swap logic for a stable so estimates match

Reply:
Looks like there is a useStablecoinPrice hook

Reply:
wondering if we should change the eth pricing to use the same hook or keep it with the way we have been previously doing it

Reply:
I think the best course would be to replace the 3rd party price fetching with the useStable hook to help consolidate. Added a tech debt ticket https://uniswaplabs.atlassian.net/browse/NFT-1062